docs/en/integrations/dbt/dbt-setup.md

```diff
@@ -0,0 +1,165 @@
+---
+sidebar_label: Setup 
+sidebar_position: 2
+description: Setup of dbt and the ClickHouse plugin
+---
+
+# Installation
+
+## dbt
+
+We assume the use of the dbt CLI for the following examples. Users may also wish to consider[ dbt Cloud](https://docs.getdbt.com/docs/dbt-cloud/cloud-overview), which offers a web-based Integrated Development Environment (IDE) allowing users to edit and run projects.
+
+dbt offers a number of options for CLI installation. Follow the instructions described[ here](https://docs.getdbt.com/dbt-cli/install/overview). At this stage install dbt-core only. We recommend the use of `pip`.
+
+```bash
+python install dbt-core
+```
+
+**Important: The following is tested under python 3.9.**
+
+## ClickHouse plugin
+
+Install the dbt ClickHouse plugin:
+
+```bash
+pip install dbt-clickhouse
+```
+
+# Prepare ClickHouse
+
+dbt excels when modeling highly relational data. For the purposes of example, we provide a small IMDB dataset with the following relational schema. This dataset originates from the[ relational dataset repository](https://relational.fit.cvut.cz/dataset/IMDb). This is trivial relative to common schemas used with dbt but represents a manageable sample:
```

Comment:
The dataset providers require to cite this paper when using their data:
https://arxiv.org/abs/1511.03086
see: [How to cite](https://relational.fit.cvut.cz/about)